pkg/msr/intel.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2012-2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file contains support functions for msr access for Linux.
+package msr
+
+const (
+	// This is Intel's name. It makes no sense: this register is present
+	// in 64-bit CPUs.
+	IntelIA32FeatureControl  MSR = 0x3A
+	IntelPkgCstConfigControl MSR = 0xE2
+	IntelFeatureConfig       MSR = 0x13
+	IntelDramPowerLimit      MSR = 0x61
+	IntelConfigTDPControl    MSR = 0x64B
+	IntelIA32DebugInterface  MSR = 0xC80
+)
+
+var Intel = []MSRVal{
```

Comment:
Intel what? This is not descriptive. What's special about these MSRs?

The name would be more accurate if it was `IntelBootLockMSRs` or something like that...

Reply:
yeah but it ends up being msr.Intel, so I don't want to call it msr.IntelBootLockMSRs

This is just "a random set of intel msr's you might care about."

and that's a hard thing to name.